src/app/modules/ledger/hw-app-symbol.js

```diff
@@ -137,6 +137,31 @@ export default class SymbolLedger {
         return signedTransaction;
     }
 
+    /**
+     * sign a Symbol Cosignature transaction with a given BIP 44 path
+     *
+     * @param path a path in BIP 44 format
+     * @param cosignatureTransaction a cosinature transaction needs to be signed
+     * @param signerPublicKey the public key of signer
+     * @param isOptinSymbolWallet if Opt-in Symbol wallet uses curve Secp256K1 else uses curve Ed25519
+     * @return a Signed Cosignature Transaction which is signed by account at path on Ledger
+     */
+    async signCosignatureTransaction(path, cosignatureTransaction, signerPublicKey, isOptinSymbolWallet) {
+        const rawPayload = cosignatureTransaction.serialize();
+        const signingBytes = cosignatureTransaction.transactionInfo.hash + rawPayload.slice(216);
```

Comment:
what's the cosignatureTransaction? 
how does this get calculated `cosignatureTransaction.transactionInfo.hash + rawPayload.slice(216);` ?